ckanext/ontario_theme/templates/internal/snippets/access_level_checkboxes.html

```diff
@@ -0,0 +1,33 @@
+{% set schema_dict = h.scheming_get_dataset_schema('dataset')['dataset_fields'] %}
+{% set name = "access_level" %}
+{%- asset "ontario_theme_scripts/ontario_theme_access_level_js" -%}
+{% with access_levels = h.scheming_field_by_name(schema_dict, name).choices %}
+  {% for levels in access_levels %}
+    {% set value = levels.get('value') %}
+    {% set count = h.get_facet_items_dict(name)|selectattr('name', '==', value)|map(attribute='count')|list %}
+    {% set display_name = h.scheming_choices_label(access_levels, value) %}
+    {% if checkboxes %}
```

Comment:
It doesn't seem like `checkboxes` can ever be False

Reply:
`checkboxes` is defined when calling the snippet (line 78 of search_form.html) `{% snippet 'snippets/access_level_checkboxes.html', checkboxes=true %}` and (line 247 of search_form.html) `{% snippet 'snippets/access_level_checkboxes.html', extras=extras %}`. Checkboxes is always false for line 247